.github/workflows/shared.yml

```diff
@@ -76,6 +76,13 @@ jobs:
             /private/var/tmp/_bazel_$(whoami)
             # Cache musl libc toolchains.
             /usr/local/*-musl
+      
+      - name: Install bazelisk
+        if: ${{ matrix.target == 'linux-aarch64' }}
+        run: |
+          wget https://github.com/bazelbuild/bazelisk/releases/download/v1.17.0/bazelisk-linux-arm64
+          chmod +x bazelisk-linux-arm64
+          sudo mv bazelisk-linux-arm64 /usr/local/bin//bazel
```

Comment:
should we checksum the bazelisk binary for good measure?

Reply:
-- also, we don't need bazelisk on macos?

Reply:
The `macos-12` runner from GitHub already comes bundled with `bazel`.